pagerenderer/vue/ui.apps/src/main/js/state.js

```diff
@@ -95,7 +95,11 @@ window.onpopstate = function(e){
         $peregrineApp.loadContent(e.state.path, false, true)
     } else {
         log.warn(e);
-        $peregrineApp.loadContent((document.location? document.location.href : null), false, true)
+        if (document.location.pathname.startsWith("/protected/")) {
```

Comment:
Could we do it here instead https://github.com/swiss-taekwondo/stkd-theme/blob/main/ui.apps/src/main/content/jcr_root/content/stkdtheme/pages/js/renderer.js#L398 ? 

Usually I'd prefer to keep peregrine repo code as generic as possible.